### ISSUE TITLE
Soulgem Scanner shows "a lazy dog" instead of gem stages and is misaligned on the bar

### ISSUE THREAD

**Issue by 2piv** [darkconsole/dse-soulgem-oven#4]
Using the scanner on PC or NPC brings up the bars that show "a lazy dog" and some other sample text. Both are distorted and misaligned. Other than that, the bar progress works properly.

**Comment by darkconsole** [comment]
nobody that has had the issue has figured out the cause, but all of them have fixed it by reinstalling pretty much everything and or a new save. i still suspect people esl'ing things they shouldn't.

**Comment by 2piv** [comment]
Do you have any hypothesis so far on why the proper names do not appear and why ESLs might trigger it? It was working fine in beta 4 but after I upgraded to beta 6 the bug appeared. Maybe I can narrow down the mods that I may have to uninstall.

**Comment by darkconsole** [comment]
one guy fixed it after reinstalling skyui and not esl'ing it which was never a good idea. someone else couldn't get that to fix it, but fixed it by reinstalling everything. the other handful of people just reinstalled everything without even trying to debug.

the only time i had it happen was when i installed one of those mods that mention something about "player voice" - for some reason those always fuck the game up pretty hard and scripts will just stop and it will lag. and i had to revert to a clean save prior to installing any mod that mentioned player voice. any of the various mods like this one. even saving the game gets all fucked up when i try these things.

![image](https://user-images.githubusercontent.com/9095897/88829271-00f97400-d192-11ea-9f8d-568b4c4d81b9.png)

**Comment by darkconsole** [comment]
so far 3 guys have fixed it by disabling "floating health bars" and i was able to confirm installing that mod broke this widget.

**Comment by 2piv** [comment]
Strange, I have never installed floating health bars before...

**Comment by ghost** [comment]
TL;DR (because I always write too much): SL Defeat, starting a new game with it + SGO beta 5 or 6 has this result for me.

I had the same issue with beta 5 and 6. In my previous game I used beta 6 and didn't had this problem. Now I'm testing different mods for a new character and _bang_, the SGO widget isn't working. So I tried to isolate the problem: I went for a new profile (MO2) with the requirements + an alternate start mod. Each time I started a new game and did the in game install for Sexlab + started SGO and tried the Scan power. No problems there, so I started adding mods a few at a time. I didn't had troubles with the widget until I got to SL Defeat. 

I'm not saying this is the only culprit, but at least for me this is a consistent way to reproduce the issue. Whenever I start a new game with Defeat enabled, the SGO widget had the "This lazy dog" text. I don't use Floating Health Bars and I've never partake in the ESLify fever. Does this make sense? is there a common thing in Floating health bars and Defeat? (I mean both adds some new UI elements, but I'm not playing Smart doc here, I don't know, all I can do is try things by pure brute force and systematic trial and error).

**Comment by reaperm1988** [comment]
Hi, I can confirm the Story of ustdont it is SL Defeat breaking the widget. But SL Defeat itself is quite a strage thing as you need to exclude the Files and the ModOrganizer from the Virus Scanner (Windows Defender) in order to not get a CTD so I suspect some things being a bit strange in this mod.

**Comment by beachbandit** [comment]
Decompiling `interface/exported/widgets/dse-soulgem-oven/meter.swf` with [jpexs-decompiler](https://github.com/jindrapetrik/jpexs-decompiler) seems to indicate that the MeterWidget has default text labels defined. 

- `This Text Label` is the default title text.
- `That lazy dog` is the default body text.

I see SL Defeat also comes with a meter.swf, but it's under its own file path, so they shouldn't interact.

**Comment by darkconsole** [comment]
widget has been redone using iwant widgets
